pkgs/tools/security/metasploit/update.sh

```diff
@@ -12,6 +12,7 @@ sed -i "s#refs/tags/.*#refs/tags/$latest\"#" Gemfile
 bundler install
 bundix
 sed -i '/[ ]*dependencies =/d' gemset.nix
+sed -i -e '$a\' gemset.nix
```

Comment:
```suggestion
```
This isn't needed, it's already fixed in `bundix`.

Reply:
Indeed, thanks.